### DISCUSSION TITLE
상태 관리 라이브러리를 활용한 API 요청 감소 및 렌더링 최적화 전략: 시험기간 트래픽 급증 시 사용자 경험 개선을 위한 성능 분석

### DISCUSSION THREAD

**Discussion by boozeal** [100-hours-a-week/discussions#22] [Deepdive]
시험기간에 몰린 많은 유저가 요청한 api가 지연 응답되는 상황이 발생.
api 요청 감소 및 렌더링 최적화를 통해 사용자 경험을 개선해야 함.
-> 상태관리 라이브러리를 통해 develop
상태관리를 통한 api 요청 수 비교
컴포넌트의 상태에 따른 렌더링 시간 분석
props의 크기 및 depth에 따른 렌더링 시간 기록
contextAPI를 사용한 경우의 렌더링 시간 비교
상태관리 라이브러리를 통해 전역으로 상태를 가져올 경우의 렌더링 시간 비교
다양한 상태관리 라이브러리별 성능 및 보일러플레이트 코드 비교
스토리지 vs 상태 관리

**Comment by startupcodekr** [comment]
상태 관리 라이브러리를 활용한 API 요청 감소 및 렌더링 최적화 전략: 시험기간 트래픽 급증 시 사용자 경험 개선을 위한 성능 분석
